doc/classes/VoxelToolTerrain.xml

```diff
@@ -17,6 +17,7 @@
 			<param index="2" name="flat_direction" type="Vector3" />
 			<param index="3" name="smoothness" type="float" default="0.0" />
 			<description>
+				Operates on a hemisphe, where the "dome" part's summit is pointed in `flat_direction`. `smoothness` determines how the flat part blends with the rounded part, with higher values producing softer more rounded edge.  
```

Comment:
Typo: "hemisphe"

`flat_direction` is actually supposed to point away from the flat side. If that's not the case I think it should be fixed.

Reply:
Oh nö, my bad!^^'

With `flat_direction` I was expecting it to work as you say, but yeah, it's flipped.

```c++
//funcs.h:353
math::sdf_smooth_subtract( //
	math::sdf_sphere(pos, center, radius), //
	math::sdf_plane(pos, flat_direction, plane_d),
	smoothness
);